tsconfig.json

```diff
@@ -11,6 +11,7 @@
   "moduleResolution": "node",
   "files": [
     "index.ts",
-    "tests/test.ts"
+    "tests/test.ts",
+    "src/utils.ts"
```

Comment:
We should take a `src/*.ts` glob next time.
http://www.typescriptlang.org/docs/handbook/tsconfig-json.html